packages/core/src/core.ts

```diff
@@ -204,3 +214,20 @@ export function saveState(name: string, value: string): void {
 export function getState(name: string): string {
   return process.env[`STATE_${name}`] || ''
 }
+
+/**
+ * Sanatizes an input into a string so it can be passed into issueCommand safely
+ * @param input input to sanitize into a string
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function sanitizeInput(input: any): string {
+  if (input == null) {
```

Comment:
suprised linter didnt complain. Can we check `=== null` and `=== undefined`?

Reply:
Thats intentional but we can make it more clear.

```
null == undefined
true
null == null
true
null === undefined
false
null === null
true
```